recipes/sortmerna/4.2.0/meta.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# FILE: meta.yaml
+# Created: Dec 08, 2019 Sun
+#
+{% set version = '4.2.0' %}
+{% set md5 = 'e684ba3893b9bc818f0763941ce1700c' %}
+
+package:
+  name: sortmerna
+  version: {{version}}
+  
+# TODO:
+#   - for now using pre-built binaries due complexities with C++17 and the dependencies.
+
+source:
+  url: https://github.com/biocore/sortmerna/releases/download/v{{version}}/sortmerna-4.2.0-Linux.tar.gz
```

Comment:
The `4.2.0` in the URL should also be replaced with `{{ version }}`.

Please add spaces around `version`: `{{ version }}` (I don’t know why, but that’s how every other recipe does it.)

Reply:
done